Scripts/DCS-BIOS/test/TestSuite.lua

```diff
@@ -13,7 +14,6 @@ package.path = "./Scripts/DCS-BIOS/lib/modules/memory_map/?.lua;" .. package.pat
 
 -- global functions that haven't been refactored yet
 BIOS = {}
-function BIOS.log(str) end -- noop
```

Comment:
I'm guessing you'll need to require lfs here. there's a mocked variant in the compile folder that should work.